editor/src/components/inspector/sections/layout-section/list-section.tsx

```diff
@@ -90,6 +91,37 @@ function getMapExpressionMetadata(
 
 export const ListSectionTestId = 'list-section'
 
+function filterVariableOption(option: VariableOption): VariableOption | null {
+  switch (option.type) {
+    case 'array':
+      return {
+        ...option,
+        children: filterKeepArraysOnly(option.children),
+        disabled: false,
+      }
+    case 'object':
+      const chilren = filterKeepArraysOnly(option.children)
```

Comment:
```suggestion
      const children = filterKeepArraysOnly(option.children)
```

Reply:
thanks, fixed the typo in https://github.com/concrete-utopia/utopia/pull/5596/commits/e69c97c1cdcb808f8f9ed6ce1b7b01f5ceb2275b